plugins/spectrum_analyzer/__init__.py

```diff
@@ -0,0 +1,404 @@
+import colorsys
```

Comment:
This is funny. We'll just make an issue to look into this later 😛 
(the fact that this file is ridiculously large)